pom.xml

```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.channelape</groupId>
 	<artifactId>shopify-sdk</artifactId>
-	<version>2.6.0-SNAPSHOT</version>
+	<version>2.7.0-SNAPSHOT</version>
```

Comment:
You can leave as 2.6.0-SNAPSHOT. That's what snapshots are for. They aren't final. It only becomes final once merged into maven central without the '-SNAPSHOT'